### DISCUSSION TITLE
Ability to configure active days (during the week, outside holidays, etc.)

### DISCUSSION THREAD

**Discussion by afillatre** [dailydotdev/daily#820] [Feature Requests]
It would be nice to be able to configure which days count for the activity streak and declare holidays. For instance, I read technical article every work day, but not during weekend, and almost never during holidays. As the result, I went from the highest rank to the lowest during my last holidays, which is pretty frustrating.

**Comment by idoshamun** [comment]
Hi @afillatre,

Thanks for the suggestion, unfortunately, at this point, we don't have plans to implement it. I understand that it can be frustrating not to be able to reach legendary every week due to different reasons, but your profile accumulates all your accomplishments every week. So you don't lose what you have achieved. I hope this makes sense.

**Reply by idoshamun** [reply]
We also have streak freeze now that you can use in case your broke your streak
